api/v1.inc

```diff
@@ -30,7 +30,11 @@ $router->add_route("GET", "v1/projects/specialdays", "api_v1_projects_specialday
 $router->add_route("GET", "v1/projects/imagesources", "api_v1_projects_imagesources");
 $router->add_route("GET", "v1/projects/:projectid/pages", "api_v1_project_pages");
 $router->add_route("GET", "v1/projects/:projectid/pagedetails", "api_v1_project_pagedetails");
-$router->add_route("GET", "v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid", "api_v1_project_page_round");
+$router->add_route("GET", "v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid", "api_v1_project_page_pageround");
+$router->add_route("GET", "v1/projects/:projectid/codepoints", "api_v1_project_codepoints");
+
+$router->add_route("POST", "v1/projects/:projectid/rounds/:roundid", "api_v1_project_round");
```

Comment:
I don't think this URL should include the round. There is no instance where a user can checkout a page for a project in a round different than what the project is currently in. Therefore the round should be inferred from the project's current round, not specified by the user.

I think a better URL for this is `v1/projects/:projectid/pages/checkout`.

Reply:
Made it a parameter.